pandas/tests/resample/test_deprecated.py

```diff
@@ -278,3 +278,36 @@ def test_resample_base_with_timedeltaindex():
 
     tm.assert_index_equal(without_base.index, exp_without_base)
     tm.assert_index_equal(with_base.index, exp_with_base)
+
+
+def test_interpolate_posargs_deprecation():
+    # GH 41485
+    df = DataFrame({"ds": ["1992-08-27 07:46:48", "1992-08-27 07:46:59"], "y": [1, 4]})
+    s = Series(
+        df.iloc[:, 1].values.reshape(-1), index=pd.to_datetime(df.iloc[:, 0].values)
+    )
```

Comment:
Is there a simpler way to create `s`?

Reply:
I think it is needed to give this logic for s, otherwise it is taking the indexes making it not operable

Reply:
can you do something like
```
idx = pd.to_datetime(["1992-08-27 07:46:48", "1992-08-27 07:46:59"])
df = DataFrame({'y': [1, 4]}, index=idx)
ser = Series([1, 4], index=idx)
```